README.md

```diff
@@ -2,15 +2,36 @@
 
 UX Aspects Seed Project provides a skeleton which can be used to build an application using [UX Aspects](https://uxaspects.github.io/UXAspects).
 
-## Getting Started
-
-##### Install with Bower
-* Install [Git](https://git-scm.com/downloads)
-* run `git clone https://github.com/UXAspects/UXAspects-Seed`
-* Install [Nodejs](https://nodejs.org/en/)
-  * Note: Node comes with [npm](https://docs.npmjs.com/getting-started/installing-node) installed but npm is updated more frequently than Node
-* Run  `npm install` from the folder where the code will be downloaded
-* Install [bower(>v1.5)](http://bower.io/#install-bower) using npm `npm install -g bower`
-* Run `bower install` in the seed project folder
-* Run `npm start`
-* Browse "http://localhost:8080/"
+# Installation
+
+1. Install [Git](https://git-scm.com/downloads)
+
+2. Clone the repository
+```bash 
+git clone https://github.com/UXAspects/UXAspects-Seed.git
+```
+3. Install [Nodejs](https://nodejs.org/en/)
+    * Note: Node comes with [npm](https://docs.npmjs.com/getting-started/installing-node) installed but npm is updated more frequently than Node
+
+4. Run npm install from the folder where the code will be downloaded
+```bash
+npm install
+```
+5. Install [bower(>v1.5)](http://bower.io/#install-bower)
+```bash
+npm install -g bower
+```
+6. Run bower install in the seed project folder
+```bash
+bower install
+```
+7. Run npm start
+```bash
+npm start
+```
+8. Browse "http://localhost:8080/"
+
+
+# License
+
+UX Aspects code is released under the MIT License. Licenses for 3rd party code included in this repository can be found in the [licenses summary](https://uxaspects.github.io/UXAspects/assets/licenses.txt).
```

Comment:
Is there any 3rd party code included the seed project? Also you linked to the UX Aspects licenses page.